python/test/unit/language/test_matmul.py

```diff
@@ -393,9 +394,12 @@ def test_mxfp(M, N, K, BLOCK_M, BLOCK_N, BLOCK_K, NUM_STAGES, nonKDim, NUM_WARPS
     b = b_f16 * b_scale_f32
     ref_out = torch.matmul(a, b).to(torch.float32)
     output = output.to(torch.float32)
-    atol = 0.0001
-    rtol = 0.0001
-    torch.testing.assert_close(ref_out, output, atol=atol, rtol=rtol)
```

Comment:
Removed rtol and increased atol: In FP8 matmul, low precision and scaling can produce large relative errors, so rtol isn’t a reliable correctness check. Absolute tolerance better reflects the expected FP8 error range.